data/hai/hai missions.txt

```diff
@@ -1658,3 +1658,121 @@ mission "Strider notification"
+					decline
+				`	Likeet gives a laugh. "It's not actually haunted, of course. Those are just stories. Do you think I'd send an entire film crew and all their expensive equipment to a place that was actually dangerous?"`
+			choice
+				`	"Alright then, I can take you there."`
+				`	"I'm not going that way. You'll need to find some other captain."`
+					decline
+			label accept
+				`	"Excellent! I'll let the crew know where to find you."`
+					accept
+	on visit
+		dialog phrase "generic cargo and passenger on visit"
+	on complete
+		event "film crew" 30
+		payment 100000 50
+		conversation
+			`You help your passengers unload all of their film equipment once you arrive on <planet>. The props seem to get more and more elaborate as they are unpacked from your ship. You note the presence of countless rubber limbs, mechanical monsters, and cans of fake blood being carried onto the set. Once you are finished, Likeet approaches you.`
+			choice
+				`	"What's the name of your movie, anyways?"`
+				`	"What's your movie about, anyways?"`
+				`	"I better get going, then."`
+					goto going
+			`	"I don't want to spoil anything, but it's a psychological horror film that explores the nature of human isolation, and the shadow that our own mortality casts over our lives." She continues to elaborate on more philosophical themes that seemingly have less and less to do with the actual film, until she notices that your eyes have long since glazed over.`
+			`	"Well, anyways, thanks for your help, captain. I'll contact you in a month once we're finished," says Likeet.`
+				accept
+			label going
+			`	"Thanks for your help, captain. I'll contact you in a month once we're finished," says Likeet.`
+				accept
+
+
+
+event "film crew"
+
+
+
+mission "Hai Film Crew [2]"
+	name "Hai Film Crew"
+	description "Go to <destination> to pick up the film crew you transported there."
+	destination "Darkrest"
+	to offer
+		has "event: film crew"
+	on offer
+		dialog
+			`Upon landing your ship, you are startled by a vaguely familiar voice coming from your intercom. You soon recognize it as Likeet, the movie director that you brought to Darkrest a month ago.`
+			`	"Captain <last>! Good to see you again! We've finished filming here and need a ride off this place. I'll see you soon." Before you can say anything, she hangs up.`
+				accept
+
+
+
+mission "Hai Film Crew [3]"
+	name "Hai Film Crew"
+	description "Take Likeet's film crew home to their studio on <destination>, so that they can process their film. Payment is <payment>."
+	minor
+	source "Darkrest"
+	destination
+		attributes hai urban
+	passengers 80
+	cargo "film equipment" 30
+	to offer
+		has "Hai Film Crew [2]: done"
+	on offer
+		conversation
+			`The first thing you notice when you land on Darkrest is the mess. It looks as if a tornado has touched down on the set, and you don't remember natural disasters being part of the movie's original script.`
```

Comment:
```suggestion
			`The first thing you notice when you land on Darkrest is the mess. It looks as if a tornado has touched down on the set, and you don't remember Likeet selling the movie as being about natural disasters.`
```
I imagine the player hasn't read the script. This change could be improved, I feel.